.github/workflows/azure-static-web-apps-ambitious-flower-024b5d90f.yml

```
@@ -0,0 +1,54 @@
+name: Azure Static Web Apps CI/CD
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)